pandas/core/api.py

```diff
@@ -25,6 +25,7 @@
                                   date_range, bdate_range)
 from pandas.tseries.tdi import TimedeltaIndex, Timedelta
 from pandas.tseries.period import Period, PeriodIndex
+from pandas.core.style import Styler
```

Comment:
let's make it an explicit import for now
I think the way to do this is actually to provide an option

eg pd.set_option('display.styler',MyStyler)

or maybe a function that generates this

eg say I want to always set some styles on display

so if I did

def mystyler(styler):
   return styler.set_precision(3).highlite_nulls()

then

pd.set_option('display.styler',mystyler)

would work